### PR TITLE
Implement webhooks for catching stripe events

### PR DESCRIPTION
**Pull request by ZeeshanAzhar23**

### CODE REVIEW

File: app/controllers/webhooks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        return
+      rescue Stripe::SignatureVerificationError => e
+        render json: {message: e}, status: 400
+        return
+      end
+      case event.type
+      when 'checkout.session.completed'
+        puts 'hello'
+       # binding.pry
+      #  return if !User.exists?(event.data.object.client_reference_id)
```
Comment (javier459) [inline app/controllers/webhooks_controller.rb]: remove commented code.

File: app/controllers/webhooks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+        fullfill_order(event.data.object)
+
+      when 'checkout.session.async_payment_succeeded'
+
```
Comment (javier459) [inline app/controllers/webhooks_controller.rb]: this is an empty when statement, remove this if not required.

File: test/controllers/features_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class FeaturesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (javier459) [inline test/controllers/features_controller_test.rb]: don't need this file.

File: test/controllers/home_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class HomeControllerTest < ActionDispatch::IntegrationTest
+  test "should get welcome" do
+    get home_welcome_url
+    assert_response :success
+  end
+
+end
```
Comment (javier459) [inline test/controllers/home_controller_test.rb]: delete this fle

File: test/controllers/plans_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PlansControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (javier459) [inline test/controllers/plans_controller_test.rb]: delete all test files

**Review by javier459 — CHANGES_REQUESTED** [review]

**Review by ZeeshanAzhar23 — COMMENTED** [review]
Kindly review the changes

**Review by javier459 — APPROVED** [review]

**Pull request merged**